usr/bin/regolith-session-init

```diff
@@ -88,6 +91,12 @@ resolve_default_config_file
 echo "Regolith is launching i3-gaps with $I3_CONFIG_FILE"
 i3 -c "$I3_CONFIG_FILE"
 
+# Run user's post logout script if script. Post logout means that the X session is closed at this point.
+if [ -f "$USER_POST_LOGOUT_SCRIPT_FILE" ] ; then
+    echo "Regolith is launching user post-logout file $USER_POST_LOGOUT_SCRIPT_FILE"
+	timeout 5 bash "$USER_POST_LOGOUT_SCRIPT_FILE"
```

Comment:
In the case that the timeout is exceeded and a user script is still running, I think it would be better to pass `-v` here so that there is a log entry that indicates that the script did not finish